runtime/compiler/trj9/env/VMJ9.cpp

```diff
@@ -1953,6 +1953,58 @@ UDATA TR_J9VMBase::thisThreadGetOSRReturnAddressOffset()
    return offsetof(J9VMThread, osrReturnAddress);
    }
 
+UDATA TR_J9VMBase::thisThreadGetGSIntermediateResultOffset()
+   {
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+   return offsetof(J9VMThread, gsParameters.intermediateResult);
+#else /* defined(OMR_GC_CONCURRENT_SCAVENGER) */
+   return 0;
```

Comment:
Shouldn't something more obviously wrong be done in this case? along with an assert?

Reply:
Good idea! I added an assert.